libs/dev-dock/backend/src/dev_dock_api.ts

```diff
@@ -109,6 +109,29 @@ function buildApi(devDockFilePath: string, machineType: MachineType) {
       return readDevDockFileContents(devDockFilePath).electionInfo;
     },
 
+    getCurrentFixtureElectionPaths(): DevDockElectionInfo[] {
+      const baseFixturePath = join(__dirname, '../../../../libs/fixtures/data');
+      return fs
+        .readdirSync(baseFixturePath, {
+          withFileTypes: true,
+        })
+        .filter((item) => item.isDirectory())
+        .map((item) => {
+          const filesInDir = fs.readdirSync(join(baseFixturePath, item.name));
+          const electionFile = filesInDir.find((file) =>
+            /^election\.json$/.test(file)
+          );
+          if (electionFile) {
+            return {
+              path: join(join(baseFixturePath, item.name), 'election.json'),
```

Comment:
I think `join` can take as many arguments as you want to join, so this could be simplified to a single `join` call